pkg/utils/apply/apply_resource_test.go

```diff
@@ -0,0 +1,196 @@
+package apply
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+
+	oamstd "github.com/oam-dev/kubevela/apis/standard.oam.dev/v1alpha1"
+	oamutil "github.com/oam-dev/kubevela/pkg/oam/util"
+)
+
+var _ = Describe("Test Server-side Applying", func() {
```

Comment:
do we still have server-side apply in this PR?

Reply:
no server-side apply anymore, let me correct the case name.